Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde = { version = "1.0.99", default-features = false, optional = true }
 pure-rust-locales = { version = "0.5.2", optional = true }
 criterion = { version = "0.3", optional = true }
 rkyv = {version = "0.7", optional = true}
+iana-time-zone = { version = "0.1.41", optional = true }
```

Comment:
By the way, this requires an MSRV bump IIRC.

Reply:
hmm true. Potentially we could resolve this by vendoring parts but that is also not ideal

Reply:
Yeah, let's just go with 1.38 for now which is still fairly conservative. Actually might be a good middle step towards 1.48, giving downstream a chance to complain?

Reply:
sounds good to me! This gets us `TryFrom` as well which is very valuable IMO

Reply:
Agreed!